src/core/time.ts

```diff
@@ -291,18 +302,23 @@ export class TimeValue {
      * Used by federates.
      */
     public toBinary(): Buffer {
-        const billion = BigInt(TimeUnit.secs);
-        let bigTime =  BigInt(this.nanoseconds) + BigInt(this.seconds) * billion;
-
-        // Ensure the TimeValue fits into a 64 unsigned integer.
-        let clampedTime = BigInt.asUintN(64, bigTime);
-        if (clampedTime != bigTime) {
-            throw new Error(`TimeValue ${this.toString()} is too big to fit into `
-                + `a 64 bit unsigned integer`);
+        let buff =Buffer.alloc(8);
```

Comment:
```suggestion
        let buff = Buffer.alloc(8);
```

Reply:
I fix this at [commit 05f87e5](https://github.com/lf-lang/reactor-ts/commit/05f87e55422607882a9b0b15f86c8e21c1cfa36d)!